pom.xml

```diff
@@ -94,6 +95,11 @@
             <version>${h2.version}</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>de.huxhorn.sulky</groupId>
+            <artifactId>de.huxhorn.sulky.ulid</artifactId>
```

Comment:
Noen grunn til ULID fremfor UUID? (Har ingenting i mot ULID altså 😄)
Kunne ULID da også bli brukt for `soknadID`?

Reply:
ULID kunne brukes for soknadID. Det var egentlig mest for å ikke sende flere lange UUID-er til frontend.

Reply:
Kan gjerne bruke ULID for min del

Reply:
Men kommer vi da til å eksponere dette til frontend?

Reply:
Ja, for å laste ned en pdf.